presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-kafka/kafka.properties

```diff
@@ -2,9 +2,16 @@ connector.name=kafka
 kafka.table-names=product_tests.read_simple_key_and_value,\
                     product_tests.read_all_datatypes_raw,\
                     product_tests.read_all_datatypes_csv,\
-                    product_tests.read_all_datatypes_json, \
-                    product_tests.read_all_datatypes_avro, \
-                    product_tests.read_all_null_avro, \
-                    product_tests.read_structural_datatype_avro
```

Comment:
Why the `Add write test configuration files` commit appears as if removing this?

i guess it's because these values had trailing spaces and now you removed them, so please fix the `Make naming regarding Kafka read tests more explicit` commit so it doesn't introduce those whitespaces in then first place